chia/wallet/nft_wallet/uncurry_nft.py

```diff
@@ -156,8 +156,9 @@ def uncurry(cls: Type[_T_UncurriedNFT], puzzle: Program) -> UncurriedNFT:
                 _, p2_args = p2_puzzle.uncurry()
                 (pubkey_sexp,) = p2_args.as_iter()
                 transfer_program_mod, transfer_program_args = transfer_program.uncurry()
-                _, _, royalty_address, royalty_percentage, _, _ = transfer_program_args.as_iter()
+                _, _, royalty_address_p, royalty_percentage, _, _ = transfer_program_args.as_iter()
```

Comment:
What is this for?

Reply:
it's a royalty address in a program form